VERSIONS.md

```diff
@@ -1,5 +1,9 @@
 # Versions
 
+## 10.1.0
+
+- Upgrade lint-staged to ^8.0.0 to support partially staged files
```

Comment:
This should probably be a major change as the `lint-staged` is a major bump too.

Reply:
Right. Ok.